bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -73,10 +72,10 @@ export class UIEvents
     else if et == "move"
       logger.debug("Registering tool: #{type} for event '#{et}'")
       if tool_view._move_enter?
-        tool_view.listenTo(@, "move:enter", tool_view._move_enter)
-      tool_view.listenTo(@, "move", tool_view["_move"])
+        tool_view.listenTo(@, "move:enter:#{id}", tool_view._move_enter)
```

Comment:
AFAIK I know anything that responding to `"move:enter"` before will also respond to this. @mattpap can you confirm?